library/java/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -118,6 +131,12 @@ String resolveTemplate(final String templateYAML, final String platformFilterTem
         .append(String.format("- &dns_fail_max_interval %ss\n", dnsFailureRefreshSecondsMax))
         .append(String.format("- &dns_query_timeout %ss\n", dnsQueryTimeoutSeconds))
         .append(String.format("- &dns_preresolve_hostnames %s\n", dnsPreresolveHostnames))
+        .append(String.format("- &h2_connection_keepalive_interval %ss\n",
+                              h2ConnectionKeepaliveIntervalSeconds))
+        .append(String.format("- &h2_connection_keepalive_idle_interval %sms\n",
```

Comment:
I dont think `ms` works automagically. Will update tomorrow.

Reply:
Only `s` works with the proto parser.